### PR TITLE
pm 53 delete user preferences on profile page

### PR DESCRIPTION
**Pull request by xHoudek**
Originally tried allowing the user to delete individual preferences, but scrapped it when it became much more difficult to implement than expected. The user already has the functionality from within the filters anyway

### CODE REVIEW

File: src/routes/ProfilePage/ProfilePage.js [hunk 1]
```diff
@@ -96,11 +108,53 @@ export default class ProfilePage extends Component {
...
+        // Simply for formatting purposes
+        return elem.value.replace(/([a-z])([A-Z])/g, '$1 $2').toLowerCase();
+      }).join(', ');
+    };
+
     return (
       <Section>
         <>
-          <h2>Saved Preferences</h2>
+          <h2>Saved Preferences <span className='btn p-2 text-primary' onClick={this.deleteAllPreferences}><FontAwesomeIcon icon='trash-alt' /></span></h2>
```
Comment (dotte-osu) [inline src/routes/ProfilePage/ProfilePage.js]: What do you think about changing `Saved Preferences` to `My pet Preferences` or something?

Comment (dotte-osu) [inline src/routes/ProfilePage/ProfilePage.js]: Maybe add more space <br> between my profile section and saved preferences section?

File: src/routes/ProfilePage/ProfilePage.js [hunk 2]
```diff
@@ -96,11 +108,53 @@ export default class ProfilePage extends Component {
...
+          <table className='table'>
+            <tbody>
+              {animalTypes ? <tr><th>Types of Animals</th><td>{animalTypes}</td></tr> : null}
```
Comment (dotte-osu) [inline src/routes/ProfilePage/ProfilePage.js]: I am being picky but what do you think about changing the text alignment to left? 
I think left alignment looks better?
image

image

**Review by dotte-osu — CHANGES_REQUESTED** [review]
The functionality works nicely! I think your code looks good. I have some tiny suggestion about styling. Let me know what you think!

File: src/routes/ProfilePage/ProfilePage.js (hunk 1 above)
Comment (xHoudek) [reply src/routes/ProfilePage/ProfilePage.js]: I think that's a good idea!

File: src/routes/ProfilePage/ProfilePage.js (hunk 2 above)
Comment (xHoudek) [reply src/routes/ProfilePage/ProfilePage.js]: How did you make the text closer together? I tried `className='align-left'` and everything stayed where it was

**Review by dotte-osu — APPROVED** [review]

**Pull request merged**